server/model/monitor.js

```diff
@@ -74,6 +74,9 @@ class Monitor extends BeanModel {
                     })
                     bean.msg = `${res.status} - ${res.statusText}`
                     bean.ping = dayjs().valueOf() - startTime;
+                    if(this.url.startsWith("https")) {
```

Comment:
```suggestion
                    if (this.url.startsWith("https")) {
```

Reply:
Oops I forgot to run the formatter. Honestly tho, I'm wondering if the monitor should store the schema as an additional variable, otherwise if in the future people want to support more protocols on top of HTTP (like ws?) we're gonna have .startsWith() everywhere. Do you have any thoughts on this?

Reply:
IMO we should add ESLint ASAP


And Monitor should have full URL and helper methods to get protocol, domain, path, query string etc